config/environments/test.rb

```diff
@@ -88,4 +88,9 @@
   routes.default_url_options = { host: ENV['FQDN'] }
 
   config.force_ssl = false
+
+  # Enable asset compilation for system tests
+  config.assets.compile = true
+  config.assets.check_precompiled_asset = false
+  config.assets.digest = false
```

Comment:
only needed for warehouse system tests
```suggestion
  if ENV['RUN_RAILS_SYSTEM_TESTS'] == 'true'
    config.assets.compile = true
    config.assets.check_precompiled_asset = false
    config.assets.digest = false
  emd
```